src/main/java/uk/gov/hmcts/reform/ccd/service/CaseDeletionLoggingService.java

```diff
@@ -63,7 +61,17 @@ public void logCaseFamilies(final List<CaseFamily> deletedLinkedFamilies,
         } else {
             logDataIfNoDeletableOrSimulatedCasesFound();
         }
-        simulatedCaseDataViewHolder.setUpData(caseDataViews);
+    }
+
+    private void logSimulatedCases() {
```

Comment:
If we populate _processedCasesRecordHolder.setSimulatedCases(simulatedCases) then this method should use CaseDataView